header.go

```diff
@@ -601,6 +601,11 @@ func (h *ResponseHeader) DisableNormalizing() {
 	h.disableNormalizing = true
 }
 
+// Disable add default contentype if no Content-Type header.
+func (h *ResponseHeader) DisableNoDefaultContentType() {
```

Comment:
This method needs to be named `DisableDefaultContentType`, the current method name implies exactly what it doesn't do.
Even better would be a method `SetNoDefaultContentType(boolean)` so you can change it instead of only setting it to true.

Reply:
@erikdubbelboer yeah, thanks your review.